.github/ISSUE_TEMPLATE/00-use-case.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Use case
+about: Describe a new use case for the specification.
+title: ''
+labels: 'use case :bulb:'
+assignees: ''
+
+---
+
+**Description**
+
+As a:
+
+- [ ] Application user/user of the configuration itself
+- [ ] API user (application developer)
+- [ ] SPI user (container or runtime developer)
+- [ ] Specification implementer
```

Comment:
What's the difference between "SPI user (container or runtime developer)" and "Specification implementer"?

Reply:
I directly copy the config template, in FT, we can just have Application user and Specification implementor. I'll make the changes.

Reply:
We have "app user", "API user" (= app developer) and "spec implementer".

"SPI user" probably doesn't make sense for FT.

Reply:
yes, I have reduced the list to Application User and Specification Implementer now.